### PR TITLE
Link fixes

### PR DESCRIPTION
**Pull request by cbush**

### CODE REVIEW

**Comment by cbush** [comment]
Warning fixes. More to come!

File: source/index.txt [hunk 1]
```diff
@@ -68,11 +68,28 @@ MongoDB Realm
...
+   collection.findOneAndReplace() </mongodb/actions/collection.findOneAndReplace>
+   collection.findOneAndDelete() </mongodb/actions/collection.findOneAndDelete>
+   collection.insertOne() </mongodb/actions/collection.insertOne>
+   collection.insertMany() </mongodb/actions/collection.insertMany>
+   collection.updateOne() </mongodb/actions/collection.updateOne>
+   collection.updateMany() </mongodb/actions/collection.updateMany>
+   collection.deleteOne() </mongodb/actions/collection.deleteOne>
+   collection.deleteMany() </mongodb/actions/collection.deleteMany>
+   collection.aggregate() </mongodb/actions/collection.aggregate>
+   collection.count() </mongodb/actions/collection.count>
```
Comment (nathan-contino-mongo) [inline source/index.txt]: i'd suggest leaving these in their own collapsible section of a nested toctree -- this section is really horrifying with everything in the index toctree, imo. Up to you though if this is how we're organizing everything at first!

File: source/logs.txt [hunk 2]
```diff
@@ -140,24 +140,6 @@ each log to extract and externally store your log history.
...
-   Trigger Logs </logs/trigger>
-   Service Logs </logs/service>
-
-.. toctree::
-   :caption: Work with Logs
-   :titlesonly:
-   :includehidden:
-   :hidden:
-
-   Access Logs With the Admin API </logs/api>
```
Comment (nathan-contino-mongo) [inline source/logs.txt]: Presumably the logging sections show now exist in the index toctree if you deleted them here?

File: source/mongodb.txt [hunk 3]
```diff
@@ -263,27 +244,14 @@ Usage Guides
...
-   :titlesonly:
-   :includehidden:
-   :hidden:
-
-   MongoDB Actions </mongodb/actions>
-   Document Schemas </mongodb/document-schemas>
-   Query Filters </mongodb/query-filters>
-   Connection Strings </mongodb/connection-strings>
-   Service Limitations </mongodb/service-limitations>
-   CRUD & Aggregation APIs </mongodb/crud-and-aggregation-apis>
```
Comment (nathan-contino-mongo) [inline source/mongodb.txt]: likewise with this toctree.

File: source/triggers.txt [hunk 4]
```diff
@@ -10,25 +10,6 @@ Triggers
...
-   Scheduled Triggers </triggers/scheduled-triggers>
-
-.. toctree::
-   :caption: Reference
-   :titlesonly:
-   :includehidden:
-   :hidden:
-
-   Trigger Snippets </triggers/trigger-snippets>
-   CRON Expressions </triggers/cron-expressions>
```
Comment (nathan-contino-mongo) [inline source/triggers.txt]: and this one

**Pull request merged**